proxy/src/telemetry/event.rs

```diff
@@ -27,9 +27,8 @@ pub struct TransportClose {
 
     pub duration: Duration,
 
-    // TODO
-    //pub rx_bytes: usize,
-    //pub tx_bytes: usize,
+    pub rx_bytes: usize,
```

Comment:
That seems more correct --- the `usize` gets cast to a `u64` when it's aggregated anyway. I'm fairly certain this is a `usize` only because that's what the commented-out code uses.